src/coprocessor/dag/rpn_expr/types/expr_builder.rs

```diff
@@ -278,6 +296,23 @@ where
     // Map pb func to `RpnFnMeta`.
     let func = fn_mapper(tree_node.get_sig(), tree_node.get_children())?;
     let args = tree_node.take_children().into_vec();
+
+    // Only Int/Real/Duration/Decimal/Bytes/Json will be decoded
+    let datums = datum::decode(&mut tree_node.get_val())?;
```

Comment:
So why not simply use datum?

Reply:
I don't think to introduce `Datum` to the RPN system is a good idea, we may deprecate `Datum` in the future.